codemod/src/plugins/replace-tokens/index.ts

```diff
@@ -0,0 +1,14 @@
+import * as postcss from "postcss";
+
```

Comment:
old plugin of replacing one token with other token. not needed after the changes we made but kept it for future use, I can remove it if you prefer